spec.emu

```diff
@@ -919,27 +917,24 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
             The host environment must perform FinishLoadingImportedModule(_referrer_, <del>_specifier_</del><ins>_moduleRequest_</ins>, _payload_, _result_), where _result_ is either a normal completion containing the loaded Module Record or a throw completion, either synchronously or asynchronously.
           </li>
           <li>
-            <p>If this operation is called multiple times with <del>the same</del><ins>two</ins> (_referrer_, <del>_specifier_</del><ins>_moduleRequest_</ins>) pair<ins>s</ins> such that:</p>
```

Comment:
I rephrased a bit this section about the requirements of `HostLoadImportedModule` to be more similar about the current spec rather than the import attributes proposal, since this proposal doesn't change the requirements for module caching.

Additionally:
- "the phase **should** not affect the identity or state of the resultant Module Record" is now a **must**, stating that the returned completion record must not be affected by the phase.
- I moved the sentence about preloading into a note, first explaining what preloading is and then saying that implementations should take into account the module phase. This is just a note because preloading is not defined anywhere, and it already happens through mechanism outside of ECMA-262 or HTML (which today I learned this makes it implementation-defined and not host-defined).

cc @lucacasonato https://github.com/tc39/proposal-import-reflection/pull/36#discussion_r1188454189